config/settings.js

```diff
@@ -28,7 +28,8 @@ const SCHEMA_PIPELINE_SETTINGS = Joi.object()
     .keys({
         metricsDowntimeJobs: SCHEMA_METRICS_DOWNTIME_JOBS,
         metricsDowntimeStatuses: SCHEMA_METRICS_DOWNTIME_STATUSES,
-        public: Joi.boolean()
+        public: Joi.boolean(),
+        groupedEvents: Joi.boolean().default(false)
```

Comment:
so each time we add a pipeline preference we need to add here ? 
@klu909 this has implication to the preference for show triggers.